examples/Kindle2Folder.rb

```diff
@@ -0,0 +1,29 @@
+require 'kindle_highlights'
+
+def sanitize_filename(filename) #Function to remove invalid symbols from filenames
+  fn = filename.split /(?<=.)\.(?=[^.])(?!.*\
+  fn.map! { |s| s.gsub /[^a-z0-9\-]+/i, '_' }
+  return fn.join '.'
+end
+
+kindle = KindleHighlights::Client.new(
+  email_address: "aaa@aaa.com",
+  password: "password"
+  )
```

Comment:
Looks like you have two extra spaces here; the `)` should line up with the first column.